### PR TITLE
Display Circle instead of Square for Dotted border

### PR DESCRIPTION
**Pull request by c-louis**
Change the Square dotted to real dot

https://github.com/dompdf/dompdf/issues/2239#issue-702636041

### CODE REVIEW

File: src/Renderer/AbstractRenderer.php [hunk 1]
```diff
@@ -474,7 +474,7 @@ protected function _get_dash_pattern($style, $width)
                 if ($width <= 1) {
                     $pattern = [$width, $width * 2];
                 } else {
-                    $pattern = [$width];
+                    $pattern = [0.4, $width * 2];
```
Comment (c-louis) [inline src/Renderer/AbstractRenderer.php:477]: Maybe those value should be change

**Comment by bsweeney** [comment]
I think this needs just a bit more work before it's ready (e.g. update the PDFLib and GD canvas instances). Going to push to the next release so I have more time to review.

File: src/Renderer/AbstractRenderer.php (hunk 1 above)
Comment (Mellthas) [reply src/Renderer/AbstractRenderer.php:477]: Should be 0 for a circle, see #2926.

**Comment by Mellthas** [comment]
@c-louis It doesn’t look like you are coming back to this, are you okay with being attributed as co-author in #2926?

**Comment by c-louis** [comment]
@Mellthas I'm glad someone had the time to make it. I am okay with being attributed as co-author.

**Pull request closed without merging**